cmd/pebble/main.go

```diff
@@ -26,6 +26,8 @@ type config struct {
 		// Require External Account Binding for "newAccount" requests
 		ExternalAccountBindingRequired bool
 		ExternalAccountMACKeys         map[string]string
+		// Configure policies to deny certain domains
+		BlockList []string
```

Comment:
Since config files are part of the "public API" of Pebble, make this name more descriptive, like "IdentifierBlocklist" or "DomainBlocklist", just in case someone wants to add an account key blocklist or similar later on.